rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -913,6 +913,26 @@ object Reduce {
         }
     }
 
+    private[this] val toUtf8Bytes: Method = new Method() {
+
+      override def apply(
+          p: Par,
+          args: Seq[Par]
+      )(implicit env: Env[Par], costAccountingAlg: CostAccountingAlg[M]): M[Par] =
+        if (args.nonEmpty) {
+          s.raiseError(ReduceError("Error: toUtf8Bytes does not take arguments"))
+        } else {
+          p.singleExpr() match {
+            case Some(Expr(GString(encoded))) =>
```

Comment:
This is really a nitpick but `encoded` is not a good name because it actually _not_ encoded :)